pkg/config/config_test.go

```diff
@@ -187,6 +187,130 @@ func TestGetStepConfigWithJSON(t *testing.T) {
 	})
 }
 
+func TestApplyAliasConfig(t *testing.T) {
+	p := []StepParameters{
+		{
+			Name: "p0",
+			Aliases: []Alias{
+				{Name: "p0_notused"},
+			},
+		},
+		{
+			Name: "p1",
+			Aliases: []Alias{
+				{Name: "p1_alias"},
+			},
+		},
+		{
+			Name: "p2",
+			Aliases: []Alias{
+				{Name: "p2_alias.deep.test"},
```

Comment:
[minor] isn't it more common to use a slash (`/`) for denoting pathes?

Reply:
@marcusholl @nevskrem I originally used `/`.
I am happy to switch that back. What do you think?

reason for going with `.` finally was that
I looked into our `handleCompatibility` (https://github.com/SAP/jenkins-library/blob/master/src/com/sap/piper/ConfigurationHelper.groovy#L71).

Reply:
@OliverNocon @marcusholl kind of awkward but in within some parts of the code we used `/` i.e.  https://github.com/SAP/jenkins-library/blob/0f3436d1a5a3b4ec7ec45b9a76ad74a5725b573a/src/com/sap/piper/ConfigurationHelper.groovy#L134 whereas for the compatibility the dot notation was used.

Since compatibility is not yet too widely spread throughout our code should we go for harmonization with `/`?

Reply:
`/` would be fine with me ...

Reply:
just switched it ...